docs/features/multiple-networks/multi-homing.md

```diff
@@ -375,16 +379,21 @@ For both ipv4 and ipv6 the following parameters are configured using DHCP or RAs
 - mtu (taken from network attachment definition)
 
 ### Configuring dns server
-By default the DHCP server at ovn-kuberntes will configure the kubernetes
+By default, the DHCP server at ovn-kubernetes will configure the kubernetes
 default dns service `kube-system/kube-dns` as the name server. This can be
 overridden with the following command line options:
 - dns-service-namespace
 - dns-service-name
 
 ## Limitations
-OVN-K currently does **not** support:
+OVN-Kubernetes currently does **not** support:
+
 - the same attachment configured multiple times in the same pod - i.e.
   `k8s.v1.cni.cncf.io/networks: l3-network,l3-network` is invalid.
+- updates to the network-attachment-definition defining the topology. The user must align all NADs pointing to the same network, 
+  then workloads must be restarted.
```

Comment:
Nice. Still, as currently written, it gives the impression that updates are **not** supported. 

Let's try to re-phrase it to something that says that updates do not take effect *until* ...

How about we remove this sentence from the list of things that "OVN-Kubernetes currently does **not** support", and add it further down, in a note or something similar ?

Something like: 
- updates to the network specification require the attached workloads restart. All the network-attachment-definitions pointing to the same network *must* have a consistent configuration, and then workloads *must* be restarted.

Reply:
I like it.